Library/Homebrew/test/cleanup_spec.rb

```diff
@@ -63,6 +63,11 @@
     end
   end
 
+  specify "::disk_cleanup_size" do
+    described_class.instance_eval("@disk_cleanup_size = 0")
```

Comment:
We can test this by writing an integration test in `test/cmd/cleanup_spec.rb` for `report_disk_usage`.